modules/insights-operator-gather-duration.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+:_content-type: PROCEDURE
+[id="insights-operator-gather-duration_{context}"]
+= Viewing Insights Operator gather durations
+
+You can view the time it takes for the Insights Operator to gather the information contained in the archive. This helps in understanding Insights Operator resource usage and troubleshooting issues with Insights Advisor.
+
+
+.Perquisites 
+
+* A recent copy of your Insights Operator archive
+
+.Procedure
+
+. From your archive, open `/insights-operator/gathers.json`.
++
+The file contains a list of Insights Operator gather operations: 
++
+[source,json]
+----
+    {
+      "name": "clusterconfig/authentication",
+      "duration_in_ms": 730,
+      "records_count": 1,
+      "errors": null,
+      "panic": null
+    }
+----
++
+`"duration_in_ms":` is the amount of time in milliseconds for each gather operation. 
```

Comment:
I know the back ticks are right, but are the quotation marks needed here?

Reply:
removed